glue_astronomy/translators/tests/test_spectrum1d.py

```diff
@@ -236,6 +236,7 @@ def test_spectrum1d_2d_data(spec_ndim):
     assert len(data.main_components) == 1
     assert data.main_components[0].label == 'flux'
     assert_allclose(data['flux'], flux.value)
+    assert data.get_component('flux').units == 'Jy'
```

Comment:
```suggestion
    assert_quantity_allclose(data['flux'], flux)
```
makes it more symmetric with `test_from_spectrum1d`